magicctapipe/scripts/lst1_magic/semi_automatic_scripts/coincident_events.py

```diff
@@ -194,6 +198,7 @@ def main():
     env_name = config["general"]["env_name"]
     LST_version = config["general"]["LST_version"]
     config_file = config["general"]["base_config_file"]
+    nice_parameter = config["general"]["nice"]
```

Comment:
maybe you can do something like this: 
```nice_parameter = config["general"]["nice"] if "nice" in config["general"] else None```
it should then set it to None if this line is missing in the config file

Reply:
I tested the solution. When I set the nice parameter, for example to 5, it works correctly: the value is read and saved in the .sh script. Then, I left 'nice:' empty and tested it again. In this case, mcp sets the 'nice' value to 0 (which I confirmed using scontrol) and does not add the line to the script.